sources/common/infoschema.go

```diff
@@ -58,18 +59,36 @@ type FkConstraint struct {
 // ProcessSchema performs schema conversion for source database
 // 'db'. Information schema tables are a broadly supported ANSI standard,
 // and we use them to obtain source database's schema information.
-func ProcessSchema(conv *internal.Conv, infoSchema InfoSchema) error {
+// numWorkers decides the parallelism while converting tables
+func ProcessSchema(conv *internal.Conv, infoSchema InfoSchema, numWorkers int) error {
 	tables, err := infoSchema.GetTables()
+	fmt.Println("fetched tables", tables)
```

Comment:
Instead of these print statements wouldn't it be better to use logger.Log.Debug

Reply:
Same comment for other print statements

Reply:
planning to take that up as a separate exercise across the file as the logs are currently not being printed to stdout and will be lost.